### ISSUE TITLE
python-trezor can't detect a device in bootloader mode for macOS

### ISSUE THREAD

**Issue by vxst** [trezor/python-trezor#98]
The macOS successfully detected the device(as a composite device):
```
Composite Device:

  Product ID:	0x0001
  Vendor ID:	0x534c
  Version:	1.00
  Serial Number:	SatoshiLabs
  Manufacturer:	TREZOR
  Location ID:	0x14200000
```
The device successfully entered the bootloader mode, which there is a screen shows the Serial No. and the loader version(1.3.0)

However, the trezorctl's get_transport function can't get the device.  The len of `HidTransport.enumerate()` is 0, and the get_transport raised a "Device not found" error.

The device can be detected when is not in bootloader mode.

**Comment by prusnak** [comment]
This is a known issue unfortunately for bootloader 1.3.0.



Please read the sticky post here which also provides workaround:



https://www.reddit.com/r/trezor/comments/5gt2co

**Comment by vxst** [comment]
I successfully connect my device after about 10 times try...
Maybe it's the problem of key-jitter elimination logic? Where is the source of the bootloader? Or is it already fixed?

**Comment by saleemrashid** [comment]
@vxst It was fixed in 1.3.1 but devices are currently shipped with 1.3.0
